meta-iot2000-example/recipes-example/iot2000setup/files/iot2000setup.py

```diff
@@ -489,7 +488,11 @@ def configureNetworkInterfaces():
 	
 
 def expandFileSystem():
+	gscreen.finish()
+	print(chr(27) + "[2J") # Clear console
```

Comment:
Handling ANSI escape sequences should be encapsulated into a (property) class or something for source code clarity like 
```
class ansicolors:
   clear = '\033[2J'
   ...

print(ansicolors.clear)
```
Something like [colorama](https://pypi.python.org/pypi/colorama) or related packages are probably too much here...